src/com/stanimiriliev/chaptertwo/auto/Automobiles.java

```diff
@@ -0,0 +1,27 @@
+package com.stanimiriliev.chaptertwo.auto;
+
+import java.text.DateFormat;
+import java.util.Date;
+
+/**
+ * Represent the common properties of the cars of several makes
+ * @author Stanimir Iliev
+ * @since 22.07.2017
+ */
+public class Automobiles {
```

Comment:
Like that I can instantiate an object from that class, You can make it abstract class, then you will not be allowed to do that : 

```
Automobiles automobiles = new Automobiles();
```
you should use some of the classes that extends it.